Ginger/GingerCoreNET/GeneralLib/Email.cs

```diff
@@ -239,10 +321,22 @@ public bool Send_SMTP()
                     }
                 }
 
+                mVE.Value = Subject;
+                string subject = mVE.ValueCalculated;
+
+                mVE.Value = Body;
+                string body = mVE.ValueCalculated;
+
                 myMail.From = fromAddress;
-                myMail.IsBodyHtml = true;
-                myMail.Subject = this.Subject.Replace('\r', ' ').Replace('\n', ' ');
-                myMail.Body = this.Body;
+                myMail.IsBodyHtml = false;
+
+                if (mHTMLReportTemplate == RunSetActionHTMLReportSendEmail.eHTMLReportTemplate.HTMLReport)
```

Comment:
what is this new field HTML Report template? 
If i am using send HTML report action but body type is free text , it will work ?